remotes/handlers.go

```diff
@@ -90,23 +90,21 @@ func MakeRefKey(ctx context.Context, desc ocispec.Descriptor) string {
 // discovered in a call to Dispatch. Use with ChildrenHandler to do a full
 // recursive fetch.
 func FetchHandler(ingester content.Ingester, fetcher Fetcher) images.HandlerFunc {
-	return func(ctx context.Context, desc ocispec.Descriptor) (subdescs []ocispec.Descriptor, err error) {
+	return func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
```

Comment:
It's unclear to me why named returns were used here; after dropping the case scope this becomes an error (due to `err` previously being shadowed).

I've elected to drop this as everything becomes more clear with explicit returns in this otherwise trivial function.